worker/pkg/workflows/datasync/activities/gen-benthos-configs/processors_test.go

```diff
@@ -276,7 +282,9 @@ func Test_buildProcessorConfigsTransformAndGenerateJavascript(t *testing.T) {
 let programOutput = undefined;
 const benthos = {
   v0_msg_as_structured: () => ({"name": "world"}),
-  v0_msg_set_structured: (val) => {
+};
+const neosync = {
+  patchMessage: (val) => {
```

Comment:
I'm gonna nitpick this but should we rename this to something else that insinuates that it is expecting a structured message, similar to how benthos is doing their `msg_set_structured`? like `patchStructuredMessage`? or something? Or is this too much?  Just wondering because it _is_ a little different than what the message itself might be? I'm guessing there is a reason they have the "structured" suffix in there.
Anyways, lmk what you think there. `patchStructuredMessage` seems like a mouthful so maybe we just leave it as `patchMessage`, just wanted to put my thought down here.

Reply:
renamed to patchStructuredMessage 👍